test/inputs/citext_input_test.rb

```diff
@@ -0,0 +1,14 @@
+# encoding: UTF-8
```

Comment:
Since we are using Ruby 2.4, could you add the special comment: `#frozen_string_literal: true`?

Reply:
@feliperenan fixed